test/images/metadata-concealment/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM BASEIMAGE
```

Comment:
for the scratch you don't have to have BASEIMAGE here and even the BASEIMAGE file also.. I build binary for all supported archs and build the docker image. you can refer the example from https://github.com/kubernetes/kubernetes/tree/master/test/images/entrypoint-tester

Reply:
Fixed.